pkg/policy/bucket-policy.go

```diff
@@ -620,6 +620,33 @@ func GetPolicies(statements []Statement, bucketName string) map[string]BucketPol
 	return policyRules
 }
 
+// GetPoliciesWithPrefix - returns a map of policies of given bucket name, prefix in given statements.
+func GetPoliciesWithPrefix(statements []Statement, bucketName, prefix string) map[string]BucketPolicy {
```

Comment:
It looks like there is little difference between GetPolicies & GetPoliciesWithPrefix, they look practically the same, can you do with this style ?

```
func GetPolicies(statements []Statement, bucketName string) map[string]BucketPolicy {
   return GetPoliciesWithPrefix(statements, bucketName, "")
}
```

Reply:
Handling both the cases in GetPolicies function itself.